src/v2/BaseLoginRouter.js

```diff
@@ -32,11 +32,21 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 
-const introspectStateToken = (settings) => {
+const introspect = (settings) => {
+  const clientId = settings.get('clientId');
   const domain = settings.get('baseUrl');
+  const scopes = settings.get('scopes');
   const stateHandle = settings.get('stateToken');
+  const redirectUri = settings.get('redirectUri');
   const version = settings.get('apiVersion');
-  return idx.start({ domain, stateHandle, version });
+  
+  if (settings.get('useInteractionCodeFlow')) {
```

Comment:
I would check stateHandle first - A customized-okta-hosted widget could be configured with this switch, but we would want to maintain the okta-hosted flow.